Example/Podfile.lock

```diff
@@ -5,7 +5,7 @@ PODS:
     - NSURL+QueryDictionary
   - Artsy+UIColors (3.1.0)
   - Artsy+UIFonts (3.2.0)
-  - Artsy-UIButtons (2.2.1):
+  - Artsy-UIButtons (2.2.2):
```

Comment:
This just happened as I was setting up emission-- I can remove if we think it'll break things.

Reply:
that's fine, not sure how that happened though